### ISSUE TITLE
Unhandled Exception: NoSuchMethodError: The method 'forEach' was called on null.

### ISSUE THREAD

**Issue by Goolpe** [zino-app/graphql-flutter#207]
I have several queries
I'm using same query code
```
Query(
        options: QueryOptions(
          document: getItems || regions,
        ),
        builder: (QueryResult result) {
        }
)

```
1. 
```
const String getItems = r'''
  query GetItems(
    $status: [OrderStatus],
    $order: [[String!]]
  ){
    getItems(
      input: {
        status: $status
      },
      order: $order
    ){
      items{
        id
      }
    }
  }
''';
```
Works fine.

2.
```
const String regions = r'''
  query Regions(
    $idList: [String!]
  ){
    regions(
      input: {
        idList: $idList
      }
    ){
      id
    }
  }
''';
```
error 
```
E/flutter (30401): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The method 'forEach' was called on null.
E/flutter (30401): Receiver: null
E/flutter (30401): Tried calling: forEach(Closure: (String, dynamic) => Null)
E/flutter (30401): #0      Object.noSuchMethod (dart:core/runtime/lib/object_patch.dart:50:5)
E/flutter (30401): #1      SplayTreeMap.addAll (dart:collection/splay_tree.dart:372:11)
E/flutter (30401): #2      new SplayTreeMap.of (dart:collection/splay_tree.dart:277:48)
E/flutter (30401): #3      new Operation 
package:graphql_flutter/…/link/operation.dart:11
E/flutter (30401): #4      QueryManager.fetchQuery 
package:graphql_flutter/…/core/query_manager.dart:71
E/flutter (30401): <asynchronous suspension>
E/flutter (30401): #5      ObservableQuery.fetchResults 
package:graphql_flutter/…/core/observable_query.dart:62
E/flutter (30401): #6      ObservableQuery.onListen ..
E/flutter (30401): #7      _runGuarded (dart:async/stream_controller.dart:805:24)
E/flutter (30401): #8      _BroadcastStreamController._subscribe (dart:async/broadcast_stream_controller.dart:213:7)
E/flutter (30401): #9      _ControllerStream._createSubscription (dart:async/stream_controller.dart:818:19)
E/flutter (30401): #10     _StreamImpl.listen (dart:async/stream_impl.dart:472:9)
E/flutter (30401): #11     _StreamBuilderBaseState._subscribe 
package:flutter/…/widgets/async.dart:135
E/flutter (30401): #12     _StreamBuilderBaseState.initState 
package:flutter/…/widgets/async.dart:109
E/flutter (30401): #13     StatefulElement._firstBuild 
package:flutter/…/widgets/framework.dart:3845
E/flutter (30401): #14     ComponentElement.mount 
package:flutter/…/widgets/framework.dart:3710
E/flutter (30401): #15     Element.inflateWidget 
package:flutter/…/widgets/framework.dart:2954
E/flutter (30401): #16     Element.updateChild 
package:flutter/…/widgets/framework.dart:2757
E/flutter (30401): #17     ComponentElement.performRebuild 
package:flutter/…/widgets/framework.dart:3746
E/flutter (30401): #18     Element.rebuild 
package:flutter/…/widgets/framework.dart:3558
E/flutter (30401): #19     ComponentElement._firstBuild 
package:flutter/…/widgets/framework.dart:3715
E/flutter (30401): #20     StatefulElement._firstBuild 
package:flutter/…/widgets/framework.dart:3863
E/flutter (30401): #21     ComponentElement.mount 
package:flutter/…/widgets/framework.dart:3710
E/flutter (30401): #22     Element.inflateWidget 
package:flutter/…/widgets/framework.dart:2954
E/flutter (30401): #23     Element.updateChild 
package:flutter/…/widgets/framework.dart:2757
E/flutter (30401): #24     ComponentElement.performRebuild 
package:flutter/…/widgets/framework.dart:3746
E/flutter (30401): #25     Element.rebuild 
package:flutter/…/widgets/framework.dart:3558
E/flutter (30401): #26     ComponentElement._firstBuild 
package:flutter/…/widgets/framework.dart:3715
E/flutter (30401): #27     ComponentElement.mount 
package:flutter/…/widgets/framework.dart:3710
E/flutter (30401): #28     Element.inflateWidget 
package:flutter/…/widgets/framework.dart:2954
E/flutter (30401): #29     Element.updateChild 
package:flutter/…/widgets/framework.dart:2757
E/flutter (30401): #30     ComponentElement.performRebuild 
package:flutter/…/widgets/framework.dart:3746
E/flutter (30401): #31     Element.rebuild 
package:flutter/…/widgets/framework.dart:3558
E/flutter (30401): #32     ComponentElement._firstBuild 
package:flutter/…/widgets/framework.dart:3715
E/flutter (30401): #33     ComponentElement.mount 
package:flutter/…/widgets/framework.dart:3710
E/flutter (30401): #34     ParentDataElement.mount 
package:flutter/…/widgets/framework.dart:4062
E/flutter (30401): #35     Element.inflateWidget 
package:flutter/…/widgets/framework.dart:2954
E/flutter (30401): #36     MultiChildRenderObjectElement.mount 
package:flutter/…/widgets/framework.dart:4981
E/flutter (30401): #37     Element.inflateWidget 
package:flutter/…/widgets/framework.dart:2954
E/flutter (30401): #38     Element.updateChild 
package:flutter/…/widgets/framework.dart:2757
E/flutter (30401): #39     ComponentElement.performRebuild 
package:flutter/…/widgets/framework.dart:3746
E/flutter (30401): #40     Element.rebuild 
package:flutter/…/widgets/framework.dart:3558
E/flutter (30401): #41     ComponentElement._firstBuild 
package:flutter/…/widgets/framework.dart:3715
E/flutter (30401): #42     StatefulElement._firstBuild 
package:flutter/…/widgets/framework.dart:3863
E/flutter (30401): #43     ComponentElement.mount 
package:flutter/…/widgets/framework.dart:3710
E/flutter (30401): #44     Element.inflateWidget 
package:flutter/…/widgets/framework.dart:2954
E/flutter (30401): #45     Element.updateChild (package:flutte
```

**Comment by Goolpe** [comment]
I found, app shows error if we don't pass variables inside query widget. I add empty brackets and it works fine

```
Query(
          options: QueryOptions(
            document: dialogs,
            variables: {}
          ),
          builder: (QueryResult resultDialog) {
})
```

**Comment by micimize** [comment]
@Goolpe can you break on exception and see what it's trying to iterate over? Also what branch/release is this on?

**Comment by Goolpe** [comment]
Fixed [#211](https://github.com/zino-app/graphql-flutter/pull/211)
